docs/user-guide/index.rst

```diff
@@ -33,6 +33,9 @@
 :ref:`Model Alignment by Direct Preference Optimization (DPO) <model-aligner-dpo>`
    DPO is a simpler alignment method compared to RLHF. DPO introduces a novel parameterization of the reward model in RLHF. This parameterization allows us to extract the corresponding optimal policy.
 
+:ref:`Model Alignment by Rejection Sampling (RS) <model-aligner-rs>`
+   RS is a simple online alignment algorithm. In RS, the policy model generates several responses. These responses are assigned a score by the reward model, and the highest scoring responses are used for SFT. 
+
```

Comment:
The following copyedit also applies to the docs/user-guide/index.rst. The copyedit isidentified by the line number in the file. I am providing the edit here since the remainder of the file cannot be edited by me.

Line 31/31  fix spelling
SteerLM 2.0 is an extension to SteerLM method that introduces an iterative training procedure to explicitly enforce the generated responses to follow the desired attribute distribution.

Reply:
Updated index.rst